sycl/doc/extensions/experimental/sycl_ext_oneapi_work_group_static.asciidoc

```diff
@@ -20,7 +20,7 @@
 == Notice
 
 [%hardbreaks]
-Copyright (C) 2023 Intel Corporation.  All rights reserved.
+Copyright (C) 2024 Intel Corporation.  All rights reserved.
```

Comment:
Don't update the copyright date.  This is the year of first publication, not the most recent year of publication.

Reply:
reverted